src/lib/pages/frontsite/PatronProfile/PaginationShow.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Pagination } from '@components/Pagination';
+import PropTypes from 'prop-types';
+import { Container } from 'semantic-ui-react';
+
+export const RenderPagination = ({
```

Comment:
I would suggest to name components with `names`, not `actions`?
```suggestion
export const Pagination = ({
```

The component name does not match the filename. We normally keep them aligned.
Do you have to create a new cmp?